integration/apps/rails-seven/docker-compose.ci.yml

```diff
@@ -30,6 +30,7 @@ services:
       # Use these to choose what is run
       - DD_DEMO_ENV_PROCESS=puma
       - DD_DEMO_ENV_FEATURES=tracing,profiling
+      - DD_INSTRUMENTATION_TELEMETRY_ENABLED=true
     expose:
```

Comment:
I made a mistake in the original implementation: this environment variable should have been set in the `app` not in the `integration-tester` originally. However because the default behavior in the past was that telemetry is enabled by default, this went unnoticed

Reply:
Got it! I see that you already broke the changes into multiple commits, which is great. 

In the future, I would additionally suggest writing something like what you did in this comment in the commit description, as this was one of those changes where you can kinda tell the what of the change, but not the why :)

"Ooops it was actually not correct before" is something that happens all the time, and that is valuable to record.

(And in general, whenever you're going around the code and spot a bug and fix it, it's relevant to call out "This was actually not correct before because [...]")

Reply:
That makes sense, will be sure to apply this in future commits!